src/types.ts

```diff
@@ -47,3 +46,12 @@ export interface WebhookEventHandlerError extends AggregateError<WebhookError> {
    */
   errors: WebhookError[];
 }
+
+declare class AggregateError<T extends Error = Error>
```

Comment:
Even though is a temporary fix, we should mention and give credit to the original repo? https://github.com/octokit/webhooks.js/pull/248#issuecomment-695092969

Reply:
oopsies sorry I missed your comment. You are right. I'll send a follow up pR